File: app/src/main/java/ch/hearc/kumoslife/Statistic.java
package ch.hearc.kumoslife;

public class Statistic
{
    private final String name;
    private int value;

    public Statistic(String name, int value)
    {
        this.name = name;
        this.value = value;
    }

    public void setValue(int value)
    {
        this.value = value;
    }

    public int getValue()
    {
        return value;
    }

    public String getName()
    {
        return name;
    }

    @Override
    public String toString()
    {
        StringBuilder builder = new StringBuilder();
        builder.append("name: ");
        builder.append(name);
        builder.append(", value: ");
        builder.append(value);
        return builder.toString();
    }
}

File: app/src/main/java/ch/hearc/kumoslife/model/statistics/Statistic.kt
package ch.hearc.kumoslife.model.statistics

import androidx.room.Entity
import androidx.room.PrimaryKey
import java.lang.StringBuilder

@Entity(tableName = "statistic")
data class Statistic(@PrimaryKey(autoGenerate = true) var id: Int, var name: String, var value: Double, var progress: Double)
{
    override fun toString(): String
    {
        val builder = StringBuilder()
        builder.append("name: ")
        builder.append(name)
        builder.append(", value: ")
        builder.append(value)
        return builder.toString()
    }
}